modules/user/server-ts/social/password/resolvers.js

```diff
@@ -2,17 +2,23 @@ import bcrypt from 'bcryptjs';
 import { pick, isEmpty } from 'lodash';
 import jwt from 'jsonwebtoken';
 import { UserInputError } from 'apollo-server-errors';
-import access from '../../access';
+import { access } from '@gqlapp/authentication-server-ts';
 import User from '../../sql';
 import settings from '../../../../../settings';
 
+const {
```

Comment:
I think this is not a good idea to make assignments like this on a module-level, it is okay to do them inside function, but not on a module-level. We loose code readability

Reply:
Done